fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfContext.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.api;
+
+/**
+ * A collection of RDF contexts that can be used to extract triples from FedoraResources
+ *
+ * @author acoburn
+ * @since Dec 4, 2015
+ */
+public enum RdfContext {
```

Comment:
Yeah, there isn't any question in my mind that there are better options than an `enum`. There is a question as to whether we just eat the `enum` for now with the good intention of revisiting it (the kernel Java API has never been very public and I have no problem grinding on it, because we personally know everyone who would care and we can just buy them a beer in Dublin for the inconvenience) or whether we stop and catch our breath for a bit and doo it really right, which is annoying and obstacular to @acoburn , who has put in a lot of work on this and deserves to see his commit numbers go up.